tests/php/View/Parsers/HTMLValueTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
```

Comment:
This is copied from https://github.com/silverstripe/silverstripe-html5/blob/2/tests/HTML5ValueTest.php